CMakeLists.txt

```diff
@@ -86,6 +86,20 @@ if(DEBUG_GLUE)
    add_definitions(-DDEBUG_GLUE=1)
 endif()
 
+
+
+if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin" AND ${CMAKE_VERSION} GREATER 3.2)
+  add_custom_target(macOS_zip COMMAND
+    ${CMAKE_COMMAND} -E tar "cfv" "nix_mx_macOS.zip" --format=zip
```

Comment:
We still have the hdf problem under linux, methinks. Hence, i limited that to mac